index.html

```diff
@@ -0,0 +1,564 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>PRICE TABLES PAGE</title>
+  <link
+    href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300;700&family=Source+Sans+Pro:wght@300;400&display=swap"
+    rel="stylesheet" />
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css" />
+  <link rel="stylesheet" href="./style/style.css" />
+</head>
+
+<body>
+  <header class="navbar">
```

Comment:
Phần header chứa nhiều thông tin em ko nên class là navbar nhé. Nếu em muốn sử dụng class của container em có thể cho vào đây cũng được để tránh phải sử dụng nhiều code.
Phần header này em nên tìm hiểu và sử dụng Flexbox để nó được tối ưu nhé
https://css-tricks.com/snippets/css/a-guide-to-flexbox/